src/winter_onboarding_2021/fleet_management_service/utils.clj

```diff
@@ -21,3 +21,7 @@
     {:flash {:error true
              :style-class "alert alert-danger"
              :message msg}}))
+
+(defn hiccup-attrs [element]
```

Comment:
Could this be named better?

Reply:
What do you think about `hiccup-attributes`?